tool_path_planner/test/utest.cpp

```diff
@@ -287,6 +329,7 @@ void runExtraRasterTest(tool_path_planner::PathGenerator& planner,
   color[1] = 0.9f;
   color[2] = 0.9f;
   viz.addPolyDataDisplay(mesh, color);
+  viz2.addPolyDataDisplay(mesh, color);
```

Comment:
Added a second display because it was too confusing to have the old waypoints mixed with the new ones.